Core/Core/Views/HorizontalMenuViewController.swift

```diff
@@ -303,3 +310,7 @@ public extension HorizontalPagedMenuDelegate {
 
     var menuItemFont: UIFont { .scaledNamedFont(.semibold16) }
 }
+
+public protocol HorizontalPagedMenuItem {
+    func itemWillBeDisplayed()
```

Comment:
knitpick, how about `menuItemWillBeDisplayed` ?  Won't hold for this if you don't like the name.

Reply:
Yeah I like it. Will do.